docs/copilot/language-models.md

```diff
@@ -26,6 +28,21 @@ The list of [models available in Copilot](https://docs.github.com/en/copilot/usi
 > [!NOTE]
 > If you are a Copilot Business or Enterprise user, your administrator needs to enable certain models for your organization by opting in to `Editor Preview Features` in the [Copilot policy settings](https://docs.github.com/en/enterprise-cloud@latest/copilot/managing-copilot/managing-github-copilot-in-your-organization/managing-policies-for-copilot-in-your-organization#enabling-copilot-features-in-your-organization) on GitHub.com.
 
+## Why use your own language model API key?
+
+Developers can now leverage models directly from Anthropic, Azure, Google, OpenAI, OpenRouter, or Ollama directly by providing a valid API key. Learn how to [use your own API in VS Code](#bring-your-own-language-model-key).
```

Comment:
Let's make it clear that we provide not RAI guarantees when using your own model.

Somewhere in this doc. Not sure on location.